recipes/gdsfactory/LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
```

Comment:
The license is already included in the tarball so it can be removed.

Reply:
fixed in https://github.com/conda-forge/staged-recipes/pull/19346/commits/de9027840e81d014190a625b259452dcbf7d6167